wrappers/python/python.cpp

```diff
@@ -116,6 +116,10 @@ PYBIND11_MODULE(NAME, m) {
             : _ptr(ptr), _itemsize(itemsize), _format(format), _ndim(ndim), _shape(shape), _strides(strides) {}
         BufData(void *ptr, size_t itemsize, const std::string &format, size_t size)
             : BufData(ptr, itemsize, format, 1, std::vector<size_t> { size }, std::vector<size_t> { itemsize }) { }
+        BufData( void *ptr       // Raw pointer
+                , size_t count  // Number of points
+                , size_t dim    // The number of floats inside a point
+        )  : BufData( ptr, sizeof(float), "@f", 2, std::vector<size_t> { count, dim }, std::vector<size_t> { sizeof(float)*dim, sizeof(float) })  { }
```

Comment:
It would be nice if this wasn't hard coded to floats.
It should be enough to add the `std::string &format` and `size_t size` parameters from the above overload and use them instead of `"@f"` and `sizeof(float)` respectively.